test_results.proto

```diff
@@ -0,0 +1,61 @@
+syntax = "proto3";
+
+package cedar;
+
+option go_package = "internal";
+
+import "google/protobuf/timestamp.proto";
+
+message TestResultsData {
+  TestResultsInfo info = 1;
+  TestResultsStorage storage = 2;
+}
+
+message TestResultsInfo {
+  string project = 1;
+  string version = 2;
+  string variant = 3;
+  string task_name = 4;
+  string task_id = 5;
+  int32 execution = 6;
+  string request_type = 7;
+  bool mainline = 8;
+}
+
+enum TestResultsStorage {
```

Comment:
does it makes sense to allow this to be user specifiable?

Reply:
You're right, it doesn't seem to make sense. We allow it to be user specifiable for logs as well which doesn't seem right either.

Reply:
But the client of this would be evergreen, so do we want to take that control away?